### PR TITLE
Join tables

### PR DESCRIPTION
**Pull request by kennankole**
* Created new tables
* Created `JOIN` tables
* Performed multiple queries as can be seen in the images below;
Screenshot from 2023-01-18 11-18-40
Screenshot from 2023-01-18 11-30-55
Screenshot from 2023-01-18 11-46-08
Screenshot from 2023-01-18 11-54-28
Screenshot from 2023-01-18 12-07-01
Screenshot from 2023-01-18 13-06-03
Screenshot from 2023-01-18 13-25-04
Screenshot from 2023-01-18 13-31-09
Screenshot from 2023-01-18 13-52-10

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -34,4 +34,28 @@ ALTER TABLE animals ADD COLUMN owner_id INT REFERENCES owners(id);
 
 
 
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(250),
+    age INT,
+    date_of_graduation DATE
+);
```
Comment (mikethreels) [inline schema.sql]: - [ ] please remember to set the `id` as primary key

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kennedy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (kennankole) [reply schema.sql]: Ok let me fix it
Thanks @mikethreels 👋🏿

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @kennankole 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks.
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**